lib/mutant/ast/meta/send.rb

```diff
@@ -56,6 +56,19 @@ def binary_method_operator?
           Types::BINARY_METHOD_OPERATORS.include?(selector)
         end
 
+        # Check if receiver is a top level constant
+        #
+        # @return [Boolean]
+        #
+        # @api private
+        def receiver_top_level_const?
+          return false unless receiver && receiver.type.equal?(:const)
```

Comment:
You can include `Mutant::AST::NodePredicates` here and replace `receiver.type.equal?` with `n_const?(receiver)`. This will make the code slightly cleaner, same for `const_parent.type.equal?(:cbase)`.